velox/vector/tests/VectorMaker.h

```diff
@@ -623,6 +623,12 @@ class VectorMaker {
 
   static VectorPtr flatten(const VectorPtr& vector);
 
+  ArrayVectorPtr createArrayVector(
+      const std::vector<vector_size_t>& offsets,
```

Comment:
I wonder if it would be easier to use this method if it allowed to specify just the offsets. It would then assume that elements appear in order and compute sizes as offset[i + 1] - offset[i].

Reply:
I will make that change. I suppose since we just assume non overlapping offsets, we should also change the Arrayvector api then and not take lengths buffer.

Reply:
The separate sizes buffer in the ArrayVector is needed to allow for out-of-order writes (used primarily when evaluating conditional expressions). However, this functionality is rarely needed when making a vector in a test.